build.rs

```diff
@@ -0,0 +1,18 @@
+use std::{env, fs, path};
+
+fn main() {

```

Comment:
I don't see anything here that the build macro won't already do automatically if you simply place the files in the various `/.windows/*` folders.